src/NzbDrone.Core/Indexers/Definitions/Rarbg/RarbgParser.cs

```diff
@@ -23,20 +23,37 @@ public RarbgParser(IndexerCapabilities capabilities)
         public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
         {
             var results = new List<ReleaseInfo>();
+            var retryTime = 60; // Time in Seconds Default to 1 Minute
+            var responseCode = (int)indexerResponse.HttpResponse.StatusCode;
 
-            switch (indexerResponse.HttpResponse.StatusCode)
+            switch (responseCode)
             {
+                case 429:
```

Comment:
For the ones that are defined in dotnet, use the enum values so it's clear what they are, or comment these out.